src/kcidb.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
Please add a shebang line to make it possible to run the script directly with `./kcidb.py` as this is handy for development even though a full command line should be used in `docker-compose.yaml`.

Please also add a copyright header like for the other scripts, for example:
```
#!/usr/bin/env python3
#
# SPDX-License-Identifier: LGPL-2.1-or-later
#
# Copyright (C) 2022 Maryam Yusuf
# Author: Maryam Yusuf <maryam.m.yusuf1802@gmail.com>
```